CLAUDE.md

```
@@ -1,423 +1,230 @@
-| 50108 | Webhook Bridge | pulse_webhook | 52100 | Search indexing & API |
-| 50109 | changedetection.io | pulse_change-detection | 5000 | Change monitoring |
-| 50210-50211 | Neo4j | pulse_neo4j | 7474/7687 | Graph database |
-| 3000 | Web UI | pulse_web | 3000 | Next.js frontend |
+| 50108 | Webhook | pulse_webhook | 52100 | Search API |
+| 50109 | changedetection | pulse_change-detection | 5000 | Monitoring |
+| 50210-11 | Neo4j | pulse_neo4j | 7474/7687 | Graph DB |
+| 50110 | Web UI | pulse_web | 3000 | Frontend |
 
-**Internal URLs (within Docker network):**
+**Internal URLs:**
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)